addons/analytic/models/analytic_account.py

```diff
@@ -96,7 +96,7 @@ def _check_company_consistency(self):
                 ('auto_account_id', 'in', [account.id for account in accounts]),
                 '!', ('company_id', 'child_of', company.id),
             ], limit=1):
-                raise UserError(_("You can't set a different company on your analytic account since there are some analytic items linked to it."))
+                raise UserError(_("No company-switching allowed for your analytic account while there are some linked analytic items around! It's a recipe for an analytical disaster!"))
```

Comment:
Replace by "You can't change the company of an analytic account having analytic items around! It's a recipe for an analytical disaster!""